googleMapsApiModule.py

```diff
@@ -19,7 +19,17 @@ def direction(origin,destination):
     logging.debug(result_url)
     webbrowser.open_new(result_url)
 
+<<<<<<< HEAD
+def timezone(search_location,timestamp):
+    address = search_location.lower()
+    address = address.replace(" ","+")
+    result_url = f'https://maps.googleapis.com/maps/api/timezone/json?location={address}&timestamp={timestamp}&key={config.key}
+    print(result_url)
+    webbrowser.open_new(result_url)
+
+=======
 @logger_config.logger 
+>>>>>>> upstream/gssoc-master
```

Comment:
Looks like merge conflict hasn't been resolved properly. These `<<<<<<` `=====` `>>>>>` sections shouldn't remain after resolving conflict. Conflict is like:

```
<<<<<<< 
changes in your pr
=======
changes in base branch
>>>>>>>
```

Resolved conflict looks:

```
Changes merged from both pr & base branch
```

(No <<<<< / ===== / >>>>> sections)

Reply:
@hritikpapa, the link I shared does say that you have to resolve conflicts before pushing changes. I think you might've skipped a step.